packages/vscode-extension/src/debug/prerequisitesHandler.ts

```diff
@@ -105,10 +105,10 @@ const ProgressMessage: { [key: string]: string } = Object.freeze({
   [Checker.M365Account]: `Checking ${Checker.M365Account}`,
   [Checker.AzureFunctionsExtension]: `Installing ${Checker.AzureFunctionsExtension}`,
   [Checker.LocalCertificate]: `Checking ${Checker.LocalCertificate}`,
-  [Checker.SPFx]: `Executing NPM Install for ${NpmInstallDisplayName.SPFx}`,
-  [Checker.Frontend]: `Executing NPM Install for ${NpmInstallDisplayName.Frontend}`,
-  [Checker.Bot]: `Executing NPM Install for ${NpmInstallDisplayName.Bot}`,
-  [Checker.Backend]: `Executing NPM Install for ${NpmInstallDisplayName.Backend}`,
+  [Checker.SPFx]: `Checking and installing NPM packages for ${NpmInstallDisplayName.SPFx}`,
+  [Checker.Frontend]: `Checking and installing NPM packages for ${NpmInstallDisplayName.Frontend}`,
+  [Checker.Bot]: `Checking and installing NPM packages for ${NpmInstallDisplayName.Bot}`,
+  [Checker.Backend]: `Checking and installing NPM packages for ${NpmInstallDisplayName.Backend}`,
```

Comment:
Update the summary wording as well.
`NPM Install for ... is executed` => `NPM packages for ... are installed`